src/userCLI/src/main/kotlin/org/vaccineimpact/api/security/AddUserToGroup.kt

```diff
@@ -0,0 +1,71 @@
+package org.vaccineimpact.api.security
+
+import org.vaccineimpact.api.db.JooqContext
+import org.vaccineimpact.api.db.Tables.MODELLING_GROUP
+import org.vaccineimpact.api.models.Scope
+import org.vaccineimpact.api.models.permissions.ReifiedRole
+import kotlin.system.exitProcess
+
+sealed class GroupDefinition
+{
+    abstract fun getGroups(db: JooqContext): Iterable<String>
+
+    class GroupList(val groups: List<String>): GroupDefinition()
+    {
+        override fun getGroups(db: JooqContext) = groups
```

Comment:
this method doesn't actually use the argument passed in; seems like this whole sealed class is overkill - all you really need is one method, getAllGroups